plugins/field-colour/README.md

```diff
@@ -64,12 +72,55 @@ Blockly.defineBlocksWithJsonArray([
 ]);
 ```
 
+### Blocks
+
+This package also provides four blocks related to the colour field. Each block has generators in JavaScript, Python, PHP, Lua, and Dart.
+- "colour_blend" takes in two colours and a ratio and outputs a single colour.
+- "colour_picker" is a simple block with just the colour field and an output.
+- "colour_random" generates a random colour.
+- "colour_rgb" generates a colour based on red, green, and blue values.
+
+You can install all four blocks by calling `FieldColour.installAllBlocks()`. This will install the blocks and all of their dependencies, including the colour field. You can choose which language generators to install by adding or removing items in the map of generators passed to `installAllBlocks`.
```

Comment:
Open question: how do I describe "choosing which language generators to install" to make it clear that they are per-block per-language generators?

Reply:
Hmm.   How about something like:

> When calling `installAllBlocks`—or any of the individual `installSomeBlock` functions—you can supply one or more `CodeGenerator` instances (e.g. `javascriptGenerator`), and the install function it will also install the correct generator function for each block for the corresponding language(s).